workspaces/copilot/plugins/copilot-backend/src/client/GithubClient.ts

```diff
@@ -98,8 +168,8 @@ export class GithubClient implements GithubApi {
     try {
       const response = await octokit.request(`GET ${path}`);
       return response.data as CopilotMetrics[];
-    } catch (error) {
-      throw ResponseError.fromResponse(error.response || error);
+    } catch (error: any) {
+      throw await ResponseError.fromResponse(error.response || error);
```

Comment:
I think we can remove the try catch here as the error returned by Octokit already contains a status property which is going to be mapped into the correct HTTP code by our error handler